src/powerset.rs

```diff
@@ -61,7 +61,7 @@ impl<I> Iterator for Powerset<I>
     fn count(self) -> usize {
         let k = self.combs.k();
         let (n, combs_count) = self.combs.n_and_count();
-        combs_count + (k + 1..=n).map(|i| checked_binomial(n, i).unwrap()).sum::<usize>()
+        remaining_for(combs_count, n, k).unwrap()
```

Comment:
I think the former implementation is better because it resembles the pattern "start with remaining combinations for `k`, and add the combination count for `k+1` throgh `n`" more closely.